notebooks/sagemaker/Deploy your own finetuned command-r-0824.ipynb

```diff
@@ -0,0 +1,423 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploy Your Own Finetuned Command-R-0824 Model from AWS Marketplace \n",
+    "\n",
+    "This sample notebook shows you how to deploy your own finetuned HuggingFace Command-R model [CohereForAI/c4ai-command-r-08-2024](https://huggingface.co/CohereForAI/c4ai-command-r-08-2024) using Amazon SageMaker. More specifically, assuming you already have the adapter weights or merged weights from your own finetuning of [CohereForAI/c4ai-command-r-08-2024](https://huggingface.co/CohereForAI/c4ai-command-r-08-2024), we will show you how to \n",
+    "1. Merge the adapter weights to the weights of the base model, if you bring only the adapter weights\n",
+    "2. Export the merged weights to the TensorRT-LLM inference engine using Amazon SageMaker\n",
+    "3. Deploy the engine as a SageMaker endpoint to serve your business use cases\n",
+    "\n",
+    "> **Note**: This is a reference notebook and it cannot run unless you make changes suggested in the notebook.\n",
+    "\n",
+    "## Pre-requisites:\n",
+    "\n",
+    "1. **Note: This notebook contains elements which render correctly in Jupyter interface. Open this notebook from an Amazon SageMaker Notebook Instance or Amazon SageMaker Studio.**\n",
+    "1. Ensure that IAM role used has **AmazonSageMakerFullAccess**\n",
+    "1. To deploy this ML model successfully, ensure that:\n",
+    "    1. Either your IAM role has these three permissions and you have authority to make AWS Marketplace subscriptions in the AWS account used: \n",
+    "        1. **aws-marketplace:ViewSubscriptions**\n",
+    "        1. **aws-marketplace:Unsubscribe**\n",
+    "        1. **aws-marketplace:Subscribe**  \n",
+    "    2. or your AWS account has a subscription to the packages for <span style=\"background-color: yellow\">Cohere Command R Bring Your Own Finetuning</span>. If so, skip step: [Subscribe to the bring your own finetuning algorithm](#1.-Subscribe-to-the-bring-your-own-finetuning-algorithm)\n",
```

Comment:
We need to add the URL of the Cohere Command R Bring Your Own Finetuning listing at here. I highlighted this part for now as a reminder.